hack/lib/constants.sh

```diff
@@ -59,6 +59,7 @@ readonly OS_GOVET_BLACKLIST=(
 	"pkg/build/vendor/github.com/docker/engine-api/client/hijack.go:[0-9]+: assignment copies lock value to c: crypto/tls.Config contains sync.Once contains sync.Mutex"
 	"cmd/cluster-capacity/.*"
 	"pkg/build/builder/vendor/.*"
+	"pkg/cmd/server/start/.*"
```

Comment:
That's bad, I don't like it, what's the reason behind it, can we fix it not to blacklist it? Especially that this is blacklisting every error and not a particular one.